torchvision/transforms/v2/_misc.py

```diff
@@ -234,27 +235,59 @@ class ToDtype(Transform):
             A dict can be passed to specify per-datapoint conversions, e.g.
             ``dtype={datapoints.Image: torch.float32, datapoints.Video:
             torch.float64}``.
+        scale:
     """
 
     _transformed_types = (torch.Tensor,)
 
-    def __init__(self, dtype: Union[torch.dtype, Dict[Type, Optional[torch.dtype]]]) -> None:
+    def __init__(self, dtype: Union[torch.dtype, Dict[Type, Optional[torch.dtype]]], scale: bool = False) -> None:
         super().__init__()
-        if not isinstance(dtype, dict):
-            dtype = _get_defaultdict(dtype)
```

Comment:
Removing this behaviour we would like to go away from `defaultdict(lambda: dtype)` ?

Reply:
Yes, this is one of the goals https://github.com/pytorch/vision/issues/7756#issuecomment-1649383342

Reply:
But should not this be done in a separate PR and everywhere it is used ?

Reply:
If this gets accepted here I was going to submit a follow-up PR to do the same changes for the `fill` parameters. Are there other places where this pattern is used?

Reply:
OK, sounds good to me.

PermuteDimensions, TransposeDimensions and `_setup_fill_arg`